fcrepo-stats-api/src/main/java/org/fcrepo/stats/api/StatsResults.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.stats.api;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @author dbernstein
+ */
+public class StatsResults {
+    @JsonProperty("mime_types")
+    private List<MimetypeStatsResult> mimetypes;
```

Comment:
The results might be clearer if the mimetype stats were nested inside of the binaries section, so:
```
{
    "all":
    {
        "resource_count": 2
    },
    "binaries":
    {
        "resource_count": 1,
        "byte_count": 265145,
        "mime_types":
	    [
	        {
	            "resource_count": 1,
	            "byte_count": 265145,
	            "mime_type": "text/csv"
	        }
	    ]
    }
}
```
So that its clear that the "mime_types" category is referring to binaries.

Also, this property should probably be named mimeTypes given the json property. Same with the class name as well I guess, and the other places where it is spelled "mimetype"